flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/Table.java

```diff
@@ -879,4 +879,117 @@
 	 * @return An OverWindowedTable to specify the aggregations.
 	 */
 	OverWindowedTable window(OverWindow... overWindows);
+
+	/**
+	 * Performs a field add operation. Similar to an SQL SELECT statement. The field expressions
```

Comment:
statement. -> statement,  ?

Reply:
I think`statement. ` is fine, is there any other reason?

Reply:
It seems to me that there are causal relationship between these two statements. They are dependent.